### PR TITLE
Feature/react native web

### PR DESCRIPTION
**Pull request by franmontiel**
Add support for React Native for the Web

### CODE REVIEW

File: example/src/App.tsx [hunk 1]
```diff
@@ -10,10 +10,28 @@ export default function App() {
...
+        // baseURL: 'https://dashboard.bugfender.com',
+        // overrideConsoleMethods: true,
+        // printToConsole: true,
+        // logUIEvents: true,
+        // registerErrorHandler: true,
+        deviceName: 'Anonymous',
+      })
+        .web({
+          version: '1.0',
+          build: '01',
```
Comment (doup) [inline example/src/App.tsx:26]: Oh, `version/build` are web specific? I find it weird.

File: src/types/sdk-options.ts [hunk 2]
```diff
@@ -15,6 +15,67 @@ export interface ISDKOptions {
...
+      enableLogcatLogging: this.nativeOptions?.enableLogcatLogging,
+    };
+
+    // Removing undefined attributes
+    Object.entries(options).forEach(([key, value]) => {
+      if (value === undefined) {
+        // @ts-ignore
+        delete options[key];
+      }
+    });
```
Comment (doup) [inline src/types/sdk-options.ts:77]: I don't know if you have an `utilities` file, maybe this could be moved there and share it with `sdk-options.ts`.

**Review by doup — APPROVED** [review]
I've added two small comments, I see it OK otherwise. 👍

File: example/src/App.tsx (hunk 1 above)
Comment (franmontiel) [reply example/src/App.tsx:26]: Native SDKs obtain that information from the builds (as all builds must declare a version)

**Pull request merged**